dask/compatibility.py

```diff
@@ -17,14 +17,18 @@
     from itertools import zip_longest
     from io import StringIO, BytesIO
     from bz2 import BZ2File
-    from gzip import GzipFile
+    from gzip import (GzipFile, compress as gzip_compress,
+            decompress as gzip_decompress)
     try:
-        from lzmaffi import LZMAFile
+        from lzmaffi import (LZMAFile, compress as lzma_compress,
+                             decompress as lzma_decompress)
     except ImportError:
-        from lzma import LZMAFile
+        from lzma import (LZMAFile, compress as lzma_compress,
+                          decompress as lzma_decompress)
     from urllib.request import urlopen
     from urllib.parse import urlparse
     from urllib.parse import quote, unquote
+    FileNotFoundError = FileNotFoundError
```

Comment:
Just puts it into the namespace so that the following works in Py3

    from .compatibility import FileNotFoundError